tutorial/part1/chapter4.md

```diff
@@ -202,7 +198,7 @@ In action, you will now have programmed the following interactive app:
 
 ### 4.3.2 Bringing everything together
 
-To end this chapter, let's tie everything together. Especially, we want to make you aware, how to use mutliple callbacks in your app. Therefore, this paragraph brings everything together we have achieved in this chapter. In order to combine both callbacks implemented above just put both of them behind each other in your code. Hereby the order doesn't matter, just make sure you have all components defined and assigned the right ids and properties. Merging everything together will give you the final app for this chapter.
+To end this chapter, let's tie everything together. Especially, we want to show an example of how to use multiple callbacks in your app. In order to combine both callbacks implemented above, we specify them in sequence in the code. The order doesn't matter, just make sure you have all components defined and assigned the right ids and properties. Merging everything together will give you the final app for this chapter.
```

Comment:
I think we can erase the last sentence because it's repetitive and doesn't add much value to the reader: "Merging everything together will give you the final app for this chapter."

Instead of this sentence, I think it could be helpful to add:
"Note that you need to insert space between the two callbacks, but there is no space between the callback decorator and the callback function."

Reply:
Technically I think that you can actually have space between the decorator and its function but it looks weird so I agree with the wording you suggested.